packages/execution-environments/package.json

```diff
@@ -23,7 +23,7 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:changelog": "yarn auto-changelog validate",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
-    "clean": "rimraf '*.tsbuildinfo' 'dist/*' 'src/__GENERATED__/'",
+    "clean": "rimraf '*.tsbuildinfo' 'dist/*' 'src/__GENERATED__/' 'coverage/*'",
```

Comment:
Why is this needed?

Reply:
It wasn't cleaned before, clean should reset the package to pristine condition